packages/vitepress-plugin-git-changelog/src/client/components/Changelog.vue

```diff
@@ -76,10 +78,18 @@ const isFreshChange = computed(() => {
 
   return differenceInDays(new Date(), lastChangeDate.value) < 1
 })
+
+const reversedCommits = computed(() => {
+  // reverse() will change the original array, so deep copy it
+  // we can also use toReversed() in order to not change original array
+  // but toReversed() has lack of compatibility
+  const temp: typeof commits.value = JSON.parse(JSON.stringify(commits.value))
```

Comment:
```suggestion
  const temp: typeof commits.value = [ ...commits.value ]
```

Reply:
oh, shallow copy is enough, since we just change the order rather the elements inside, thx.

Reply:
Would you please click the *Commit suggestion** if you accept my suggestion?